Platon/app/src/main/java/com/cmpe451/platon/adapter/WorkspaceListAdapter.kt

```diff
@@ -85,6 +85,7 @@ class WorkspaceListAdapter(private val data: ArrayList<WorkspaceListItem>, priva
         this.notifyDataSetChanged()
     }
     fun submitElements(list: List<WorkspaceListItem>){
+        data.clear()
```

Comment:
i remember, data.clear() prevents pagination at some respect, can you double check.

Reply:
we do not have pagination for workspaces, so it was doubling the data